quantize.py

```diff
@@ -521,21 +530,22 @@ def quantize(
     percdamp: float = .01,
     blocksize: int = 128,
     label: str = '',
+    device: str = 'cuda',
 ) -> None:
     assert checkpoint_path.is_file(), checkpoint_path
 
-    device = 'cpu'
     precision = torch.bfloat16
 
     print("Loading model ...")
+    print(f"Target device={device}")
     t0 = time.time()
 
     with torch.device('meta'):
         model = Transformer.from_name(checkpoint_path.parent.name)
 
     checkpoint = torch.load(str(checkpoint_path), mmap=True, weights_only=True)
     model.load_state_dict(checkpoint, assign=True)
-    model = model.to(dtype=precision, device=device)
+    model = model.to(dtype=precision, device='cpu')
```

Comment:
don't we need the model over on GPU if device='cuda' ? Or do we keep it on CPU and depend on that `   weight.to(torch.bfloat16).to(device=self.device), self.groupsize, self.inner_k_tiles` to get it there?  (That seems inconsistent with typical pytorch practice?)

Reply:
Wait a second.... I missed this in my review.

Lemme back out this commit.